### ISSUE TITLE
Templates

### ISSUE THREAD

**Issue by curran** [unhcr/dataviz-styleguide#24]
We'll want to have templates using various technologies to create visualizations that follow this style guide.

Technologies:
 * Excel
 * R (ggplot) - see #23 
 * D3.js
 * DC.js
 * PowerBI
 * Tableau
 * Adobe Illustrator (does the really work for creating data-driven graphics?)

Visualization Types (first pass, ambitious, subject to change):
 * Bar Chart
 * Grouped Bar Chart
 * Stacked Bar Chart
 * Horizontal Bar Chart
 * Horizontal Grouped Bar Chart
 * Horizontal Stacked Bar Chart
 * Histogram
 * Line Chart
 * Multi-Line Chart
 * Area Chart
 * Stacked Area Chart
 * StreamGraph - started in [streamgraph-explorer - streamGraph.js](https://github.com/curran/streamgraph-explorer/blob/master/src/streamGraph.js)
 * Population Pyramid
 * Bubble Map
 * Choropleth Map

As we work on these, let's create new issues for each (Technology, Visualization Type) pair that we want to target.

**Comment by Edouard-Legoupil** [comment]
Something to consider could be also to include (at least for D3.js Viz), a template for the layout. For instance, I tried to consistently use the [Yeti bootstrap](https://bootswatch.com/yeti/) template for the last online dashboard

**Comment by Edouard-Legoupil** [comment]
For D3, might be good to explicitly add [dc.js](https://dc-js.github.io/dc.js/) in that list. It's easier than d3 to get quickly something out. Maybe one could build some easy to adapt dashboard template for this where one would just need to change the data and configure a setting file - see some [ideas for 4W dasbaord here](https://github.com/protection-cluster/dashboard-monitoring-template).

**Comment by Edouard-Legoupil** [comment]
Would also suggest to create a few Power BI Desktop templates for the redundant use case (starting with the 4 main ones: Activities Tracking, Population Profile, Financial Tracking, Protection Monitoring). We have more and more IMO using PowerBI and no standardised look and feel so far..

**Comment by vidonne** [comment]
@Edouard-Legoupil This is exactly the idea behing that project is that we will provide basic generic representations and some templates. Having it open-source will help us extend the number of templates as if people like you really at ease with R can develop one based on the recommendations. The same go for everything Power-Bi, Tableau, etc what ever technologies people want to use.

**Comment by curran** [comment]
Great ideas! Just added the following to the list

 * DC.js
 * PowerBI
 * Tableau

**Comment by curran** [comment]
Closing in favor of using Trello

**Comment by Edouard-Legoupil** [comment]
Trello? Where is it?

**Comment by curran** [comment]
We opted to use Trello to track issues for this project.

The board is here https://trello.com/b/vbkBr11S/data-viz-guideline-sprint

One unfortunate side effect of this (that I think we did not think through) is that the project is no longer really open, as only the UNHCR team can see the issues, prioritization, and who's working on what.

@vidonne I wonder if we could open the Trello board up so @Edouard-Legoupil and others could see it as well?
